src/ray/raylet/worker.cc

```diff
@@ -32,6 +33,8 @@ pid_t Worker::Pid() const { return pid_; }
 
 Language Worker::GetLanguage() const { return language_; }
 
+int Worker::Port() const { return port_; }
```

Comment:
Okay for now, but I would think about whether the port really needs to be exposed to the caller. For instance, it would be better if we can abstract away the communication from the raylet to the worker so that the `NodeManager` doesn't need to know whether the worker has a grpc server or not.

Reply:
btw The worker would always have a grpc server as long as it's not a driver.